### ISSUE TITLE
Anyone using Rollup? 

### ISSUE THREAD

**Issue by Jaikant** [ReactBangalore/meta#4]
Hey guys, 

Anyone using [Rollup](https://rollupjs.org/).
Would be great to have a talk on this, to understand its pros and cons vis-a-vis webpack.

**Comment by akiran** [comment]
I haven't used it, but I read that "Rollup is more suitable for libraries and webpack for projects"

**Comment by siddharthkp** [comment]
This is the best idea ever 😄 

Let the audience decide what do they want to learn, and hopefully someone will know and give a talk about it 😄

**Comment by akiran** [comment]
Yes, I got some messages requesting react-router v4 talk and Handons react-native workshop

May be we can maintain list of interesting topics and help speakers pick topics
